packages/protocol/test/common/metatransactionwallet.ts

```diff
@@ -328,7 +336,15 @@ contract('MetaTransactionWallet', (accounts: string[]) => {
               .executeTransactions(
                 transactions.map((t) => t.destination),
                 transactions.map((t) => t.value),
-                ensureLeading0x(transactions.map((t) => trimLeading0x(t.data)).join('')),
+                ensureLeading0x(
+                  transactions
+                    .map((t) => {
+                      const data = trimLeading0x(t.data) // @ts-ignore
```

Comment:
why do we need the @ts-ignore?

Reply:
for the annoying reason of a shadow variable, which IIIIII think it perfectly fine(!!!) but it failed the build, so I will have to change this anyway.